src/uct/ib/rdmacm/rdmacm_listener.c

```diff
@@ -65,7 +65,8 @@ UCS_CLASS_INIT_FUNC(uct_rdmacm_listener_t, uct_cm_h cm,
 
     if (rdma_bind_addr(self->id, (struct sockaddr*)saddr)) {
         status = ((errno == EADDRINUSE) || (errno == EADDRNOTAVAIL)) ?
-                 UCS_ERR_BUSY : UCS_ERR_IO_ERROR;
+                 UCS_ERR_BUSY :
+                 ((errno == ENODEV) ? UCS_ERR_NO_DEVICE : UCS_ERR_IO_ERROR);
```

Comment:
1. need switch/case here?
2. why need to silence ENODEV?

Reply:
also, what about tcp sockcm?

Reply:
> also, what about tcp sockcm?

I dont think that tcp has such issue, if the interface has an IP address TCP should be abale to work with it, that is not true for rdmacm